posts/2022-09-29-22.0.0.11-beta.adoc

```diff
@@ -0,0 +1,174 @@
+
+You can enable the feature by adding `servlet-6.0` to your `server.xml`:
+
+[source, xml]
+----
+  <featureManager>
+    <feature>servlet-6.0</feature>
+  </featureManager>
+----
+
+For more information, refer to the link:https://jakarta.ee/specifications/servlet/6.0/[Jakarta Servlet 6.0 Specification and Javadocs].
+
+
+// https://github.com/OpenLiberty/open-liberty/issues/22423
+[#mail]
+=== Simpler Jakarta Mail code compilation (Jakarta Mail 2.1)
+
+Jakarta Mail 2.1 support allows your Jakarta EE 10 applications to send, read, and edit E-Mail from mail servers via the Jakarta Mail APIs.
+
+The biggest change to the Jakarta Mail 2.1 specification is the removal of the direct dependency of the specification APIs on the reference implementation. This means that, in certain cases, there is no longer a need to compile your Jakarta Mail code against both the specification and the reference implementation (provided you aren't making any direct calls to the reference implementation in your code).
+
+To facilitate this improvement, Jakarta Mail 2.1 also introduces the link:https://jakarta.ee/specifications/mail/2.1/apidocs/jakarta.mail/jakarta/mail/util/streamprovider[jakarta.mail.util.StreamProvider] utility interface, which can be easily obtained with the `jakarta.mail.Session.getStreamProvider()` method. You can then use the `StreamProviders` instance to obtain encoded/decoded streams for any of the encoding/decoding methods that Jakarta Mail 2.1 supports.
+
+You can enable the feature by adding `mail-2.1` to your `server.xml`:
+
+[source, xml]
+----
+  <featureManager>
+    <feature>mail-2.1</feature>
+  </featureManager>
+----
+
+For more information about the Jakarta Mail 2.1 specification, check out the link:https://eclipse-ee4j.github.io/mail/[Jakarta Mail] project website.
+
+// https://github.com/OpenLiberty/open-liberty/issues/22115
+[#connectors]
+=== Ensure correctness of data types (Jakarta Connectors 2.1)
+
+Jakarta Connectors standardizes how connectors to various enterprise information systems interact with the container and applications.  Jakarta Connectors 2.1 enables you to use the latest release of the link:https://github.com/eclipse-ee4j/jca-api[Jakarta resource specification] in your application.
+
+The main update to Jakarta Connectors in this minor feature version update is the addition of Java generics support to the `jakarta.resource.cci.ResourceFactory`, `jakarta.resource.cci.IndexedRecord`, and `jakarta.resource.cci.MappedRecord` classes.  This helps avoid compiler warnings and help ensure correctness of data types during development time.
```

Comment:
Consider changing "This helps avoid compiler warnings and help ensure correctness" to "This helps avoid compiler warnings and **also** help**s** ensure correctness"

Reply:
updated to "... and ensure correctness..."